triton-inference-server.spec

```diff
@@ -0,0 +1,43 @@
+### RPM external triton-inference-server 1.12.0
+%define branch master
+%define github_user NVIDIA
+
+Source: git+https://github.com/%{github_user}/triton-inference-server.git?obj=%{branch}/v%{realversion}&export=%{n}-%{realversion}&output=/%{n}-%{realversion}.tgz
+BuildRequires: cmake
+Requires: openssl opencv protobuf grpc curl python py2-wheel py2-setuptools py2-grpcio-tools
+
+%prep
+
+%setup -n %{n}-%{realversion}
+
+%build
+
+# remove config required in cmake
+sed -i 's/find_package(CURL CONFIG REQUIRED)/find_package(CURL REQUIRED)/' ../triton-inference-server-1.12.0/src/clients/c++/library/CMakeLists.txt
```

Comment:
can you please replace `triton-inference-server-1.12.0` with `%{n}-%{realversion}` here and in next line too